api/ruby/rendering-data-as-graphs/server.rb

```diff
@@ -10,8 +10,8 @@ class MyGraphApp < Sinatra::Base
     #  CLIENT_SECRET    = ENV['GITHUB_CLIENT_SECRET']
     # end
 
-    CLIENT_ID = ENV['GITHUB_CLIENT_ID']
-    CLIENT_SECRET = ENV['GITHUB_CLIENT_SECRET']
+    CLIENT_ID = ENV['GH_GRAPH_CLIENT_ID']
```

Comment:
Setting these back to their prior values because this project has [official documentation](https://developer.github.com/v3/guides/rendering-data-as-graphs/) that we should avoid conflicting with.

Reply:
Reverted this in 097a124 as Warden/`sinatra_auth_github` was still expecting the `GITHUB_CLIENT_ID`/`GITHUB_CLIENT_SECRET` environment variables.

Reply:
So does the documentation need updated, then?

Reply:
@kylemacey perhaps. I'm not 100% sure why the `GH_GRAPH_CLIENT_ID`/`GH_GRAPH_CLIENT_SECRET_ID` wasn't getting passed to Warden. It still wanted `GITHUB_CLIENT_ID`/`GITHUB_CLIENT_SECRET` to be set.